solutions/devsprint-cecilia-saraiva-2/FinanceAppTests/AccountTests.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  AccountTests.swift
+//  FinanceAppTests
+//
+//  Created by Gabriel de Castro Chaves on 23/08/22.
+//
+
+import XCTest
+@testable import FinanceApp
+
+class AccountTests: XCTestCase {
+
+    func testFormattedAgency() {
+        
+        let account = Account(agency: "1", account: "")
+        
+        let formattedAgency = account.formattedAgency()
+        
+        let test = "Agency: 1"
```

Comment:
[sugestao] talvez possamos dar um nome mais significativo aqui, que tal expectedResult or algo do tipo?
```suggestion
        let expectedResult = "Agency: 1"
```